### PR TITLE
fix: add symfony/property-info in dependency

### PR DESCRIPTION
**Pull request by flochntrl**
| Q             | A
| ------------- | ---
| Bug fix?      | yes

Add the symfony/property-info in depency because we have a break in prod env.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "license": "EUPL-1.2",
     "require": {
         "php": "^7.4 || ^8.0",
-        "sylius/sylius": "^1.9"
+        "sylius/sylius": "^1.9",
+        "symfony/property-info": "^5.4"
```
Comment (oallain) [inline composer.json:16]:
```suggestion
        "symfony/property-info": "^4.4|^5.4"
```

**Review by oallain — CHANGES_REQUESTED** [review]
Can you also sign CLA ?
Thanks @flochntrl

**Review by Jibbarth — CHANGES_REQUESTED** [review]
I think in your case, the package [phpdocumentor/reflection-docblock](https://packagist.org/packages/phpdocumentor/reflection-docblock) should also be added, because it's not a direct dependency of [symfony/property-info](https://packagist.org/packages/symfony/property-info)

**Comment by oallain** [comment]
Clove in favor of #48

**Pull request closed without merging**